docs/detections/alerts-run-osquery.asciidoc

```diff
@@ -28,7 +28,7 @@ NOTE: The host associated with the alert is automatically selected. You can spec
 [role="screenshot"]
 image::images/setup-query.png[width=80%][height=80%][Shows how to set up the query]
 
-. (Optional) Expand the **Advanced** section to view or set {kibana-ref}/advanced-osquery.html[mapped ECS fields] included in the results from the live query.
+. (Optional) Expand the **Advanced** section to view or set {kibana-ref}/osquery.html[mapped ECS fields] included in the results from the live query.
```

Comment:
```suggestion
. (Optional) Expand the **Advanced** section to view or set {kibana-ref}/osquery.html[mapped ECS fields] included in the results from the live query.
```

Reply:
Thanks for filing this @melissaburpo ! I think we'll need to make one more minor tweak to point users towards the "Map result fields to ECS" section. I'll commit this to check whether the change I suggested works.